Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ansible = "*"
```

Comment:
Originally this copied over the version constraint from `requirements.txt`. However, that's not quite how `pipenv` wants to be used:

> Warning: Your Pipfile now contains pinned versions, if your requirements.txt did.
We recommend updating your Pipfile to specify the "*" version, instead.

Instead the recommend relying on the `Pipfile.lock`.